Ciphers/AffineCipher.js

```diff
@@ -44,8 +65,13 @@ function isCorrectFormat (str, a, b) {
     throw new TypeError('Argument str should be String')
   }
 
+  if (!(GetEuclidGCD(a, 26) === 1)) {
+    throw new Error(a + ' is not coprime of 26 !!!')
```

Comment:
You don't need the exclamation marks.

Reply:
i replace it with the import coprimecheck

Reply:
I was referring to the three exclamation marks in the error message.